src/redux/slices/example.js

```diff
@@ -5,18 +5,28 @@ export default function exampleResponse() {
         id: '550e8400-e29b-41d4-a716-446655440000',
         type: 'meal',
         attributes: {
-          time: 123345534,
+          childId: '5ee62461-adb8-4618-a110-06290a787223',
+          time: '12:54',
           description: 'Yogurt with strawberries',
```

Comment:
time is a long, not formatted like this. We'll need to do the formatting on the frontend

Reply:
example: 1625817827